src/commands/move.ts

```diff
@@ -162,21 +162,21 @@ export class ScrollUpCommand extends EmacsCommand {
 
     public execute(textEditor: TextEditor, isInMarkMode: boolean, prefixArgument: number | undefined) {
         const repeat = prefixArgument === undefined ? 1 : prefixArgument;
-        return createParallel(repeat, () =>
-            repeat > 1 ?
-            vscode.commands.executeCommand("cursorMove", {
+
+        if (repeat === 1) {
+            return vscode.commands.executeCommand(isInMarkMode ? "cursorPageDownSelect" : "cursorPageDown");
+        }
+
+        return vscode.commands.executeCommand(
```

Comment:
Thanks so much for reviewing this! I love this extension. The only thing is I like the scroll commands to always scroll the editor, not just when mark is out of the viewport. I use the commands instead of the mouse.

Something like this seems to be working for me:
```js
        return vscode.commands.executeCommand(
            "cursorMove",
            {
                to: "down",
                by: "wrappedLine",
                value: repeat,
                select: isInMarkMode,
            },
        ).then(() => vscode.commands.executeCommand("editorScroll", {
            to: "down",
            by: "wrappedLine",
            value: repeat
        }));
```

Let me know if this is okay

Reply:
@GuyShane Hello, sorry for my late response.
I released a new version v0.12.0 including a modification of C-v behavior which may suits this your requirement.
Please check it by setting `emacs-mcx.strictEmacsMove` config to be true (Ref: https://github.com/tuttieee/vscode-emacs-mcx#configs ).